examples/lines_bars_and_markers/markevery_prop_cycle.py

```diff
@@ -5,8 +5,9 @@
 
 This example demonstrates a working solution to issue #8576, providing full
 support of the markevery property for axes.prop_cycle assignments through
-rcParams. Makes use of the same list of markevery cases from
-https://matplotlib.org/examples/pylab_examples/markevery_demo.html
+rcParams. Makes use of the same list of markevery cases from the
+:ref:`markevery demo
+<sphx_glr_gallery_lines_bars_and_markers_markevery_demo.py>`.
```

Comment:
I prefer links of the form
```
:doc:`markevery demo </gallery/lines_bars_and_markers/markevery_demo>`
```
which are shorter, directly map to the file path, and link to the page itself instead of an anchor slightly below the top.